test/sql/SqlTest.cpp

```diff
@@ -29,17 +29,35 @@ TEST(SqlTest, wrongCredentialsShouldThrowException)
 
 TEST(SqlTest, tableEmptyGetAllProductsShouldReturnEmptyVector)
 {
-    Sql sql; //TODO Insert correct credentials
+    Sql sql;
     EXPECT_EQ(sql.getAllProducts().size(), 0);
 }
 
+TEST(SqlTest, productsDontExistGetProductsBeginningWithShouldReturnEmptyVector)
+{
+    Sql sql;
+    EXPECT_EQ(sql.getProductsBeginningWith("test").size(), 0);
```

Comment:
mozna zrobic EXPECT_TRUE(sql.getProductsBeginningWith("test").empty())

To co zrobiles tez jest ok, ale raczej do sprawdzania, czy cos jest puste, korzystamy z metody empty(), zamiast porownywac size() do z 0